source/h2_decoder.c

```diff
@@ -769,6 +796,12 @@ static int s_state_fn_frame_settings_i(struct aws_h2_decoder *decoder, struct aw
  */
 static int s_state_fn_frame_push_promise(struct aws_h2_decoder *decoder, struct aws_byte_cursor *input) {
 
+    if (decoder->aws_h2_decoder_settings.enable_push == 0) {
+        /* treat the receipt of a PUSH_PROMISE frame as a connection error of type PROTOCOL_ERROR.(RFC-7540 6.5.2) */
+        DECODER_LOG(ERROR, decoder, "PUSH_PROMISE is invalid, the seeting for enable push is 0");
```

Comment:
"seeting" -> "setting"
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
I already installed it, as you told me before. But it just did report this error and the one above. Lol.